gradle/verification-metadata.xml

```diff
@@ -10136,6 +10136,12 @@
             <sha256 value="e6ca607c3fce03e8fa38af3374ce1f8bb098e316e8bf6f6d27331360feddb1c1" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.scala-lang" name="scala-reflect" version="2.13.0">
+         <artifact name="scala-reflect-2.13.0.jar">
+            <md5 value="cbbeec15c3444405743aba51dfe4fd0d" origin="Generated by Gradle"/>
+            <sha256 value="aef72378abbc8b05cfdc7144fc0e55904dd87bf46ce2e973ae667dc3ed2fcee8" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="org.scala-lang" name="scala-reflect" version="2.13.10">
          <artifact name="scala-reflect-2.13.10.jar">
```

Comment:
I think this could be a possible `scala-reflect` dependency conflict

Reply:
It doesn't seem to cause any issues. From what I understand, it allows the artefact to have either of the generated checksums.

Reply:
Perfect, then it makes sense! :)